app/cdap/components/OngoingDataExploration/mock.ts

```diff
@@ -0,0 +1,60 @@
+export const mockData = [
```

Comment:
addd expected interface

also move mock to a folder and keep index and types inside the folder